src/components/views/rooms/RoomHeader.tsx

```diff
@@ -118,6 +129,47 @@ export default function RoomHeader({
 
     const askToJoinEnabled = useFeatureEnabled("feature_ask_to_join");
 
+    const toggleCallButton = (
+        <Tooltip label={isViewingCall ? _t("voip|minimise_call") : _t("voip|maximise_call")}>
+            <IconButton onClick={toggleCall}>
+                <VideoCallIcon />
+            </IconButton>
+        </Tooltip>
+    );
+    const joinCallButton = (
+        <Button size="sm" onClick={videoCallClick}>
+            Join
+        </Button>
+    );
+    const startCallButton = (
+        <Tooltip label={!videoCallDisabledReason ? _t("voip|video_call") : videoCallDisabledReason!}>
+            <IconButton
+                disabled={!!videoCallDisabledReason}
+                aria-label={!videoCallDisabledReason ? _t("voip|video_call") : videoCallDisabledReason!}
```

Comment:
```suggestion
        <Tooltip label={videoCallDisabledReason ?? _t("voip|video_call")}>
            <IconButton
                disabled={!!videoCallDisabledReason}
                aria-label={videoCallDisabledReason ?? _t("voip|video_call")}
```

This is equivalent, right?